common/djangoapps/student/views.py

```diff
@@ -206,6 +208,13 @@ def index(request, extra_context=None, user=AnonymousUser()):
     # Insert additional context for use in the template
     context.update(extra_context)
 
+    # getting all the programs from catalog
+    if configuration_helpers.get_value("DISPLAY_PROGRAMS_ON_MARKETING_PAGES",
+                                       settings.FEATURES.get("DISPLAY_PROGRAMS_ON_MARKETING_PAGES")):
+        programs_list = get_programs_data(user)
```

Comment:
We should add a comment here to indicate that the index.html template included in this repository will not display programs. You need to create a custom theme that overrides index.html in order to display programs. Maybe add some background about why this is currently the case.